turbo.json

```diff
@@ -36,7 +36,6 @@
     },
     "//#manypkg": {},
     "//#knip": {},
```

Comment:
I think this is the issue (it won't let me add a comment on a deleted line):

```suggestion
    "//#knip": {},
    "//#typesync": {
      "inputs": [
        "**/package.json",
        ".typesyncrc"
      ]
    },
```

Reply:
Checked it out, `pnpm typesync` correctly fails locally, so I think it's just a turbo-misconfiguration.

Reply:
Turbo isn't involved at all. I removed the related turbo task and command doesn't run with `turbo run` anymore.

Reply:
Here's the output I get:

<details><summary>Here's the output I get:</summary>
<p>

```sh
$ pnpm typesync                                

> sheriff-monorepo@ typesync /Users/dukese01/Developer/sheriff
> typesync --dry=fail

»  TypeSync v0.13.3
»  —— DRY RUN — will not modify file ——
✖  Typings changed; check failed.
»  

📦 sheriff-monorepo — package.json (no new typings added)


📦 sheriff-webservices — apps/sheriff-webservices/package.json (no new typings added)


📦 docs-website — apps/docs-website/package.json (no new typings added)


📦 config-validation-playground — apps/config-validation-playground/package.json (1 new typings added)
└─ + @types/lodash

📦 cli-playground — apps/cli-playground/package.json (no new typings added)


📦 tsconfig — packages/tsconfig/package.json (no new typings added)


📦 @sheriff/utils — packages/sheriff-utils/package.json (no new typings added)


📦 @sherifforg/types — packages/sheriff-types/package.json (no new typings added)


📦 @sherifforg/create-config — packages/sheriff-create-config/package.json (no new typings added)


📦 @sherifforg/constants — packages/sheriff-constants/package.json (no new typings added)


📦 @sherifforg/cli — packages/sheriff-cli/package.json (no new typings added)


📦 eslint-config-sheriff — packages/eslint-config-sheriff/package.json (1 new typings added)
└─ + @types/lodash

✨  Run typesync again without the --dry flag to update your package.json.
 ELIFECYCLE  Command failed with exit code 1.
``` 

</p>
</details>

Reply:
i get this

![output](https://github.com/user-attachments/assets/c79dd3ef-82a0-4f68-9767-933e521d3884)

Reply:
Typesync 0.14.0 just out, should fix it.